src/GeneticSharp.Domain/OperatorsStrategies/IOperatorsStrategy.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using GeneticSharp.Domain.Chromosomes;
+using GeneticSharp.Domain.Crossovers;
+using GeneticSharp.Domain.Mutations;
+using GeneticSharp.Domain.Populations;
+
+namespace GeneticSharp.Domain
+{
+    public interface IOperatorsStrategy
```

Comment:
Please, add documentation to the interface explaining its purpose.

Reply:
Done